app/components/sort_component/sort_component.html.slim

```diff
@@ -1,22 +1,37 @@
-= tag.div(**html_attributes) do
-  - if sort.count > 4
-    = form_for sort_form, as: "", url: path.call(url_params), method: :get, data: { controller: "form", "hide-submit": true } do |f|
-      = f.govuk_collection_select :sort_by,
-        sort,
-        :by,
-        :display_name,
-        label: { text: t("jobs.sort_by.label"), size: "s", class: "govuk-label inline govuk-!-margin-right-2" },
-        form_group: { classes: "govuk-!-margin-bottom-2" },
-        data: { action: "change->form#submitListener" }
+  = tag.div(**html_attributes) do
+    - case display_type
+    - when "inline-select"
+      = form_for sort_form, as: "", url: path.call(url_params), method: :get, data: { controller: "form", "hide-submit": true } do |f|
+        = f.govuk_collection_select :sort_by,
+          sort,
+          :by,
+          :display_name,
+          label: { text: t("jobs.sort_by.label"), class: "govuk-!-padding-right-1" },
+          form_group: { classes: "govuk-!-margin-bottom-4 sort-form-container" },
+          data: { action: "change->form#submitListener" },
+          classes: "min-width-auto"
```

Comment:
:raising_hand: Hard to be sure in a PR diff, but it seems the whole block go an extra tab/spacing that shouldn't be there?